composer.lock

```diff
@@ -241,257 +241,33 @@
         }
     ],
     "packages-dev": [
-        {
```

Comment:
The whole lock file must not change, because there is no need to update dependencies for this purpose. Instead after modifying the json file, use `composer update --lock` to prevent composer warnings while keeping existing dependencies.